### ISSUE TITLE
Treat Javadoc tags that don't start lines as plain text [phab:D1309]

### ISSUE THREAD

**Issue by kythebot** [google/kythe#1697]
_Created by **zarko** at 2017-02-07 00:09:43_: 

This addresses the particular issue where authors with
email addresses show up unmarked; for example:

```
Feigns translucence.
author cctower.eas (Cincinnatus C.)
```

would appear as

```
Feigns translucence. tower.eas (Cincinnatus C.)

Author
  * cc
```

**Comment by kythe** [comment]
_**zarko** wrote at 2017-02-08 23:26:30_:

ping

**Comment by kythe** [comment]
_**fromberger** wrote at 2017-02-08 23:35:24_:

I meant to accept this, but apparently missed the button the first time. Let's try again. :)

**Comment by kythe** [comment]
_**fromberger** accepted this revision at 2017-02-08 23:35:24_

**Comment by kythe** [comment]
_**zarko** closed via commit 3e06c6b0d47e7a9b280bd2cf10699c8a1292ab03 at 2017-02-08 23:37:10_
